index.js

```diff
@@ -34,12 +34,12 @@ async function resolveGlobbedPath (entries, cwd) {
     expandDirectories: false
   })
 
-  const paths = Object.keys(resolvedEntries.reduce((result, entry) => {
+  const paths = Object.values(resolvedEntries.reduce((result, entry) => {
     // Globby yields unix-style paths.
     const normalized = path.resolve(entry)
 
     if (!result[normalized]) {
-      result[normalized] = true
+      result[normalized] = entry
     }
 
     return result
```

Comment:
This makes the `normalized` variable rather redundant, no? So we could just skip this entire `reduce` and just return `resolvedEntries` straight up?

Makes me wonder why its originally been added there at all 🤔

Reply:
As I read it, the reason you resolve the path is to make all the requires absolute and then use the `result` hash to dedupe them?

Reply:
That's true, good catch.

> Because the code passes them to globby twice, the second time round nothing is matched because **it doesn't understand forward slashes** as path separators

I'm thinking that we maybe instead should ensure that whatever globby is given is always using the right slashes? Adding a `entry.replace('\', '/')` to https://github.com/dependency-check-team/dependency-check/blob/ad715e54b416bf2cae06c4328d3291eb6b125122/index.js#L26-L35 ? What do you think? Then we will keep the OS-type path separators in the result of the method, which  should be preferable considering the other file interactions?

Reply:
It seems to work with replacing the slashes before passing the entries to globby, I've updated the PR.